components/DropzoneComponent.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { DropzoneOptions, useDropzone } from 'react-dropzone';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+interface DropzoneComponentProps extends DropzoneOptions {
+  onChange: (file: File | null) => void;
+}
+
+const DropzoneComponent = ({
+  onChange,
+  accept,
+  maxSize,
+  minSize,
+  multiple,
+  ...rest
+}: DropzoneComponentProps) => {
+  const onHandleDrop = (acceptedFiles: File[]) => {
+    onChange(acceptedFiles.length > 0 ? acceptedFiles[0] : null);
+  };
+
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept,
+    maxSize,
+    minSize,
+    multiple,
+    onDrop: onHandleDrop,
+    ...rest,
+  });
+
+  const style = {
```

Comment:
Static functions and objects are recommended to be declared outside of the component.